examples/example_uq_mcmc.py

```diff
@@ -0,0 +1,229 @@
+"""
```

Comment:
I put my comments for this file at auto_examples.

Reply:
I addressed the comments by updating examples/example_uq_mcmc.py in the next commit.